README.md

```diff
@@ -30,9 +30,13 @@ installed with header files (and a C++ compiler of course):
 * Brew: `brew install taglib`
 * MacPorts: `sudo port install taglib`
 
-Then do:
+Then install the latest taglib-ruby 1.x by running:
 
-    gem install taglib-ruby
+    gem install taglib-ruby --version '< 2'
+
+If you are using TagLib 2.0.1 or higher, you need to install taglib-ruby 2.x instead:
+
+    gem install taglib-ruby --version '>= 2'
```

Comment:
I looked around a bit and quite a few distributions are still on TagLib 1.x. The `main` README.md is a kind of landing page so I think we shouldn't assume people reading it are on taglib-ruby 2.x.